src/snowcli/__about__.py

```diff
@@ -1,3 +1,3 @@
 from __future__ import annotations
 
-VERSION = "1.1.1-dev"
+VERSION = "1.1.1"
```

Comment:
What will happen if users will install it, then we'll introduce some fixes and release the final 1.1.1?

I think, user's pip will see that they already have snowcli 1.1.1 and won't install the version with fixes.

Reply:
They will have to force-reinstall. Adding suffix will require change in already tested version which is not a good practice.